annotation-utils/src/main/java/eu/europeana/annotation/utils/parse/AnnotationLdParser.java

```diff
@@ -609,6 +626,78 @@ private Body parseBody(JSONObject valueObject, MotivationTypes motivation) throw
 		return body;
 	}
 
+	/**
+	 * @param body
+	 * @param streetAddress
+	 * @throws JsonParseException
+	 */
+	private void setVcardStreetAddress(Body body, String streetAddress) throws JsonParseException {
```

Comment:
The first parameter must be of type VcardAddressBody the instanceOf checking and the exception throwing can be removed from the method

Reply:
removed instanceOf check and throwing exception

Reply:
there is till a todo, please implement this as well :
// TODO: separate specific fields for parsing the whole specific
				// object